<!DOCTYPE html>
<html lang="en">

  <head>
    <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-61354538-3"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'UA-61354538-3');
  </script>


    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">

    <script defer src="https://use.fontawesome.com/releases/v5.0.7/js/all.js"></script>

    <title>SCANGEN: Single-Cell Cancer Genomics workshop at ISMB 2020</title>

    <!-- Bootstrap core CSS -->
    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="css/scrolling-nav.css" rel="stylesheet">

    <link rel="icon" href="figs/dna_small.jpg">

  </head>

  <body id="page-top">

    <!-- Navigation -->
    <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top" id="mainNav">
      <div class="container">
        <a class="navbar-brand js-scroll-trigger" href="#page-top">SCANGEN</a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarResponsive">
          <ul class="navbar-nav ml-auto">
            <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#summary">Summary</a>
            </li>
            <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#submissions">Call for submissions</a>
            </li>
            <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#invitedspeakers">Invited speakers</a>
            </li>
            <!-- <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#program">Program</a>
            </li> -->
            <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#travelaward">Travel award</a>
            </li>

            <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#workshopinfo">Workshop Info</a>
            </li>
            <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#organizers">Organizers</a>
            </li>
            <li class="nav-item">
              <a class="nav-link js-scroll-trigger" href="#previous">Previous meetings</a>
            </li>
            </li>
          </ul>
        </div>
      </div>
    </nav>


    <header style="background-image: url(figs/background.jpg); background-size: cover;">
      <div class="container text-center text-white" >
        <h2>SCANGEN: Single-Cell Cancer Genomics</h2>
        <p class="lead">Workshop at ISMB 2020</p>
      </div>
    </header>




    <section id="summary">
      <div class="container">
        <div class="row">
          <div class="col-lg-8 mx-auto">
            <h4>Summary</h4>
            <p>The second workshop on single-cell cancer genomics (SCANGEN) will take place <b>virtually</b> on <b>Thursday, July 16th</b> at
            <a href="https://www.iscb.org/ismb2020">ISMB 2020</a>.
              The workshop aims to cover computational and statistical methods, models, and applications in single-cell cancer genomics.
            </p>
            <p>
              Contributed talks and posters are now <b>open for submission</b>. The deadline is <b>June 5th</b>. See <a href="#submissions">call for submissions</a> for more details.
            </p>
            <h6><u>Scope:</u></h6>
            
            <ul><p>
              <li><b>Modelling cancer evolution:</b> phylogenetic inference from single-cell data; inference of evolutionary processes from single-cell data; identifying single-cell cancer signatures; inference of fitness from single-cell analysis of population dynamics</li>
              <li><b>Integrative analyses of multi-modal data:</b> modelling of joint measurement assays (such as G&T-seq); relating and interpreting measurements from different technologies</li>
              <li><b>Scalable inference at the single-cell level:</b> scalable statistical inference for single-cell data using methods such as stochastic optimization; computational tools for dealing with large single-cell datasets</li>
              <li><b>Interactions and perturbations at the single-cell level:</b> how cells interact with their microenvironment; how cells respond to and resist chemotherapeutic interventions; how transcriptional programming and clonal selection are affected by genomic perturbations such as CRISPR</li>
            </p> </ul>
            
          </div>
        </div>
      </div>
    </section>

    <!-- <section id="program">
        <div class="container">
          <div class="row">
            <div class="col-lg-8 mx-auto">
              <h4>program</h4>
              <p>
              <table class="table table-striped">
                  <thead>
                    <tr>
                      <th>Start</th>
                      <th>End</th>
                      <th>Presenter</th>
                      <th>Title</th>
                    </tr>
                  </thead>
                  <tbody>
                      <tr><td>8:35</td><td>8:40</td><td>&nbsp;</td><td>Introduction / Welcome</td></tr>
                      <tr><td>8:40</td><td>9:00</td><td>Jean Fan, Harvard University (invited)</td><td> Integrated genetic and transcriptional analysis at the single-cell level</td></tr>
                      <tr><td>9:00</td><td>9:20</td><td>Kieran O'Neill, BC Cancer Agency</td><td>Determing the Mechanism of 5-Azacytidine Response in Myeloid Malignancies Using Single-cell DNA Methylation Sequencing Paired With Flow Cytometry</td></tr>
                      <tr><td>9:20</td><td>9:40</td><td>Sombeet Sahu, Mission Bio</td><td>Methods for Identifying Tumor Heterogeneity and Rare Subclones in Single Cell DNA Sequence Data</td></tr>
                      <tr><td>10:15</td><td>10:20</td><td>&nbsp;</td><td>Welcome Back</td></tr>
                      <tr><td>10:20</td><td>10:40</td><td>Chris Yau, University of Birmingham (invited)</td><td>Scalable Bayesian Tensor Factorization for single-cell Genomics</td></tr>
                      <tr><td>10:40</td><td>11:00</td><td>Huidong Chen, Massachusetts General Hospital Research Institute</td><td>STREAM: Single-cell Trajectories Reconstruction, Exploration And Mapping of omics data</td></tr>
                      <tr><td>11:00</td><td>11:20</td><td>Sahar Ansari, University of Michigan</td><td>Leveraging transcription factor networks to identify cell types from single-cell transcriptomes</td></tr>
                      <tr><td>11:20</td><td>11:40</td><td>Xun Zhu, University of Hawaii Cancer Center</td><td>Granatum X: A community engaging and flexible scRNA-Seq analysis environment connecting tool developers to bench scientists</td></tr>
                      <tr><td>11:40</td><td>12:00</td><td>Wei Vivian Li, University of California, Los Angeles</td><td>scImpute: An accurate and robust imputation method  for single-cell RNA-seq data</td></tr>
                      <tr><td>12:00</td><td>12:20</td><td>Davis McCarthy, EMBL-EBI (invited)</td><td>An approach to studying clonal cell populations using bulk exome and single-cell RNA sequencing data</td></tr>
                      <tr><td>12:20</td><td>12:40</td><td>Simone Ciccolella, Universit√† degli studi Milano Bicocca</td><td>Inferring Cancer Progression from Single Cell Sequencing while allowing loss of mutations</td></tr>

                      <tr><td></td><td>&nbsp;</td><td>&nbsp;</td><td><b>Lunch and posters</b></td></tr>
                      <tr><td>14:00</td><td>14:20</td><td>Florian Markowetz, CRUK Cambridge Institute (invited)</td><td>Inferring cancer evolution and mutational processes</td></tr>
                      <tr><td>14:20</td><td>14:40</td><td>Mohammed El-Kebir, University of Illinois at Urbana-Champaign</td><td>SPhyR: Tumor Phylogeny Estimation from Single-Cell Sequencing Data under Loss and Error</td></tr>
                      <tr><td>14:40</td><td>15:00</td><td>Violeta Beleva Guthrie, Johns Hopkins University</td><td>Single-cell sequencing analysis of pancreatic cancer precursor lesions reveals convergent evolution and polyclonal origins</td></tr>
                      <tr><td>15:00</td><td>15:20</td><td>Haoyun Lei, Carnegie Mellon University</td><td>Deconvolution of tumor copy number data using bulk and single-cell sequencing data</td></tr>
                      <tr><td>15:20</td><td>15:40</td><td>Katharina Jahn, ETH Zurich (invited)</td><td>Modelling tumour evolution from single-cell sequencing data</td></tr>
                      <tr><td>15:40</td><td>16:00</td><td>Bojan Losic, Icahn Institute for Muliscale Biology</td><td>Regulatory underpinnings of intra-tumor heterogeneity in HCC liver cancer</td></tr>
                      <tr><td>16:00</td><td>16:20</td><td>Sumon Ahmed, University of Manchester</td><td> Investigating Haematopoietic Stem Cells development by single cell sequencing</td></tr>
                      <tr><td>16:20</td><td>16:40</td><td>Rui Kuang, University of Minnesota Twin Cities</td><td>A multitask learning approach for clustering multiple single cell RNA-seq datasets </td></tr>
                  </tbody>
              </table>
              </p>
          </div>
        </div>
      </div>
    </section> -->


    <section id="submissions">
      <div class="container">
        <div class="row">
          <div class="col-lg-8 mx-auto">
            <h4>Call for submissions</h4>
            <p>A call for submissions for contributed talks (15 minutes + 5 minutes for questions) <b>is now open</b>. </p> 
            <p>The deadline for submissions is <b>June 5th 17:00 in a time zone of your choice.</b></p>
            <p>To submit an abstract for consideration, visit <a href="https://easychair.org/conferences/?conf=ismb2020abstracts">ISMB2020 on easychair</a> and select <b>SCANGEN: Single-cell cancer genomics (Special Session)</b> as the track.</p>
          </div>
        </div>
      </div>
    </section>

    <section id="invitedspeakers">
      <div class="container">
        <div class="row">
          <div class="col-lg-8 mx-auto">
            <h4>Invited speakers</h4>
            <p>
              <ul>
                <li>Smita Krishnaswamy, Yale</li>
                <li>Sohrab Shah, Memorial Sloan Kettering Cancer Center</li>
                <li>Claudia Kleinman, McGill</li>
              </ul>
            </p>
          </div>
        </div>
      </div>
    </section>

    <section id="travelaward">
      <div class="container">
        <div class="row">
          <div class="col-lg-8 mx-auto">
            <h4>Travel award for early career researchers</h4>
            <p>
              Given the virtual nature of the meeting, in lieu of travel awards a number of free one-day registrations will be available for selected presentations. More details to follow.
            </p> 
          </div>
        </div>
      </div>
    </section>


    <section id="workshopinfo">
      <div class="container">
        <div class="row">
          <div class="col-lg-8 mx-auto">
            <h4>Workshop info</h4>
            <p>The workshop is co-located at <a href="https://www.iscb.org/ismb2020">ISMB 2020</a>. 
              Due to the ongoing Covid-19 situation, the entire conference will now be held virtually, on <b>Thursday July 16th</b>.
              For information on registration, see the main ISMB conference page.</p>
          </div>
        </div>
      </div>
    </section>



    <section id="organizers">
      <div class="container">
        <div class="row">
          <div class="col-lg-8 mx-auto">
            <h4>Organizers</h4>
            <p>
                <span class="fa-li"></span><i class="fas fa-user"></i></span> <a href="https://www.camlab.ca">Kieran R Campbell</a>, Principal Investigator, Lunenfeld-Tanenbaum Research Institute & University of Toronto 
              </p>
              <p><i class="fas fa-envelope"></i> kierancampbell _at_ lunenfeld _dot_ ca 
              </p>
          </div>
        </div>
      </div>
    </section>


    <section id="previous">
      <div class="container">
        <div class="row">
          <div class="col-lg-8 mx-auto">
            <h4>Previous meetings</h4>
            <p>
              <ul class="fa-ul">
                <li>
                  <a href="http://www.scangen.org/index-2018.html">
                  SCANGEN 2018, Chicago
                  </a>
                </li>
              </ul>
            </p>
          </div>
        </div>
      </div>
    </section>

    <!-- Footer -->
    <footer class="py-5 bg-dark">
      <div class="container">
        <p class="m-0 text-center text-white">Copyright &copy; 2018-2020</p>
      </div>
      <!-- /.container -->
    </footer>

    <!-- Bootstrap core JavaScript -->
    <script src="vendor/jquery/jquery.min.js"></script>
    <script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

    <!-- Plugin JavaScript -->
    <script src="vendor/jquery-easing/jquery.easing.min.js"></script>

    <!-- Custom JavaScript for this theme -->
    <script src="js/scrolling-nav.js"></script>

  </body>

</html>
